src/include/stat.h

```diff
@@ -605,9 +605,14 @@ struct __wt_connection_stats {
     int64_t checkpoint_prep_recent;
     int64_t checkpoint_prep_total;
     int64_t checkpoint_state;
-    int64_t checkpoint_scrub_target;
-    int64_t checkpoint_scrub_time;
+    int64_t checkpoint_scrub_dirty_target;
```

Comment:
Why this is renamed dirty target? scrub itself contains the meaning that we are doing dirty eviction.